tests/PerformanceHarness/example_trx_generator_run.md

```diff
@@ -0,0 +1,97 @@
+
+## Command
+Sleep is needed otherwise subjective transaction may reject transactions as too soon.
+```
+CHAIN_ID=$(cleos --url $HTTP_URL get info | grep chain_id | cut -d:  -f2 | sed 's/[ ",]//g')
```

Comment:
I'm not suggesting that you need to change this, but if you're open to using `jq` you can do this shorter via `cleos get info | jq -r .chain_id`